frigate/ptz/onvif.py

```diff
@@ -51,7 +51,9 @@ def __init__(
                             cam.onvif.port,
                             cam.onvif.user,
                             cam.onvif.password,
-                            wsdl_dir=Path(find_spec("onvif").origin).parent / "../wsdl",
+                            wsdl_dir=str(
+                                Path(find_spec("onvif").origin).parent / "wsdl"
+                            ).replace("dist-packages/onvif", "site-packages"),
```

Comment:
Hm, ok. That breaks for me, given that the onvif package lands in site-packages in my environment.